src/components/modal/modal.scss

```diff
@@ -5,6 +5,7 @@
  *
  * @prop --calcite-modal-padding:Specifies the padding of the modal.
  * @prop --calcite-scrim-background: The component's semi-transparent background color.
+ * @prop --calcite-modal-content-background: Specifies the background color of the component's content.
```

Comment:
nitpick - "of content placed in the `content` slot" or something along those lines?